actor/builtin/miner/miner_test.go

```diff
@@ -326,40 +325,75 @@ func TestMinerSubmitPoSt(t *testing.T) {
 	st, vms := core.CreateStorages(ctx, t)
 
 	ancestors := th.RequireTipSetChain(t, 10)
-
 	origPid := th.RequireRandomPeerID(t)
 	minerAddr := createTestMiner(t, st, vms, address.TestAddress, []byte("my public key"), origPid)
+	proof := th.MakeRandomPoStProofForTest()
+
+	miner := state.MustGetActor(st, minerAddr)
+	minerBalance := miner.Balance
+	owner := state.MustGetActor(st, address.TestAddress)
+	ownerBalance := owner.Balance
+
+	firstCommitBlockHeight := uint64(3)
+	secondProvingPeriodStart := LargestSectorSizeProvingPeriodBlocks + firstCommitBlockHeight
+	lastSubmission := secondProvingPeriodStart + LargestSectorSizeProvingPeriodBlocks + LargestSectorGenerationAttackThresholdBlocks
```

Comment:
Did you intend to add `LargestSectorSizeProvingPeriodBlocks` twice to obtain lastSubmission?

Reply:
No, I intended to pay the maximum fee before the submission would be rejected entirely.